docs/topics/sgx/device-plugin-before-k8s-1-17.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sgx-device-plugin
+  namespace: kube-system
+  labels:
+    app: sgx-device-plugin
+spec:
+  selector:
+    matchLabels:
+      app: sgx-device-plugin
+  template:
+    metadata:
+      labels:
+        app: sgx-device-plugin
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: beta.kubernetes.io/instance-type
+                operator: In
+                values:
+                - Standard_DC2s
+                - Standard_DC4s
+                - Standard_DC1s_v2
+                - Standard_DC2s_v2
+                - Standard_DC4s_v2
+                - Standard_DC8_v2
+              - key: beta.kubernetes.io/os
+                operator: In
+                values:
+                - linux 
+      containers:
+      - name: sgx-device-plugin
+        image: mcr.microsoft.com/aks/acc/sgx-device-plugin:1.0
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - name: device-plugin
+          mountPath: /var/lib/kubelet/device-plugins
+        - name: dev-sgx
+          mountPath: /dev/sgx
+        securityContext:
+          privileged: true
+      volumes:
+      - name: device-plugin
+        hostPath:
+          path: /var/lib/kubelet/device-plugins
+      - name: dev-sgx
+        hostPath:
+          path: /dev/sgx
```

Comment:
nitpick: new line

Reply:
Is this a standard?

Reply:
I think in general having a single new line at the end of a file is good practice, and often needed.